tests/integration/integration_config.yml.template

```diff
@@ -47,5 +47,11 @@ sc_config:
       use_auth: false
       auth_user: ""
       auth_password: ""
+    oidc:
+      uuid: oidcconfig_uuid
+      client_id: "d2298ec0-0596-49d2-9554-840a2fe20603"
+      shared_secret: "~0lI5n-_8dNbWmrrqqTE1v1iBcfO4__jfx"
+      config_url: https://login.microsoftonline.com/76d4c62a-a9ca-4dc2-9187-e2cc4d9abe7f/v2.0/.well-known/openid-configuration
+      scopes: "openid+profile"
```

Comment:
Does this need to be added to github variables `CI_CONFIG_HC_IP50`?

Reply:
yes, this info should not be public.

Reply:
I changed  / removed all secret values from the examples, tests and configs.
But I cannot add them to GITHUB since I am not an admin. Do you have access @justinc1 ?

Reply:
I added them.